src/commands/api-mesh/__tests__/log-get-bult.test.js

```diff
@@ -0,0 +1,226 @@
+const fs = require('fs');
```

Comment:
typo in the file name `log-get-bult.test.js`

Reply:
Lol. Great catch